roles/foreman_installer/tasks/install.yml

```diff
@@ -25,7 +25,7 @@
       {{ foreman_installer_command }} {{ (foreman_installer_verbose|bool) | ternary("-v", "") }}
       {{ (foreman_installer_no_colors|bool) | ternary("--no-colors", "") }}
       {{ (foreman_installer_version is version('3.4', '<') and foreman_installer_disable_system_checks|bool) | ternary("--disable-system-checks", "") }}
-      {{ (foreman_installer_version is version('3.4', '>=') and foreman_installer_disable_system_checks|bool) | ternary("--tuning development", "") }}
+      {{ (foreman_installer_version is version('3.4', '>=') and foreman_installer_disable_system_checks|bool and foreman_installer_scenario in ['katello']) | ternary("--tuning development", "") }}
```

Comment:
You use `in` because you foresee other scenarios doing the same?

Reply:
Aye, thinking of bringing tuning to other scenarios so leaving the door open for now.